app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        print("Invalid command format.")
+        return
+    source_file, target_file = parts[1], parts[2]
+    if source_file == target_file:
+        print("Source and target files are the same. Nothing to copy.")
```

Comment:
It must do nothing in case the user is trying to copy file to file with the same name